Extension/.scripts/code.ts

```diff
@@ -0,0 +1,40 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { spawnSync } from 'child_process';
+import { verbose } from '../src/Utility/Text/streams';
+import { $args, $root, $scenario, brightGreen, checkFile, gray, green, pwd } from './common';
+
+import { resolve } from 'path';
+import { getTestInfo } from '../test/common/selectTests';
+import { install, options } from "./vscode";
+
+export { install, reset } from './vscode';
+
+export async function main() {
+    let ti = (await getTestInfo($scenario));
```

Comment:
Why is this surrounded by `()` same as the await below. If they're unnecessary can you remove them? I've never seen these type of extra parens around an awaited expression before.

Reply:
I have enabled an eslint rule to fix these; it hits 48 files for 220 instances of it.

Reply:
Oh? Is that okay? I didn't notice other cases. Is that a more general "extra paran" lint rule?

Reply:
It's a generic 'no extra parens' rule - but it's the same thing everywhere -- extra parens where they aren't necessary.

Reply:
Sometimes extra parentheses can improve readability, so I'm a little concerned about that. Ideally, the linter would only apply it in cases in which there is no potentially readability benefit. By readability benefit, I mean cases where the order of operation may be confused, such as in `&& ||` expressions, etc.